<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Jo" ignored="0" name="Test" tests="4" failures="1" timestamp="2022-04-20T12:22:02 IST" time="0.037" errors="0">
  <testcase name="login" time="0.006" classname="com.TestNG_FEATURES.Login_demo">
    <skipped/>
  </testcase> <!-- login -->
  <testcase name="login" time="0.001" classname="com.TestNG_FEATURES.Login_demo">
    <skipped/>
  </testcase> <!-- login -->
  <testcase name="login" time="0.0" classname="com.TestNG_FEATURES.Login_demo">
    <skipped/>
  </testcase> <!-- login -->
  <testcase name="login" time="0.001" classname="com.TestNG_FEATURES.Login_demo">
    <failure type="java.lang.AssertionError" message="expected [jothe] but found [jothi]">
      <![CDATA[java.lang.AssertionError: expected [jothe] but found [jothi]
at com.TestNG_FEATURES.Login_demo.login(Login_demo.java:15)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Test -->
